lib/bitcoin/BitcoinProcessor.ts

```diff
@@ -283,6 +313,47 @@ export default class BitcoinProcessor {
     return this.serviceInfoProvider.getServiceVersion();
   }
 
+  /**
+   * Gets the lock information for the specified identifier (if specified); if nothing is passed in then
+   * it returns the current lock information (if one exist).
+   *
+   * @param lockIdentifier The identifier of the lock to look up; should be null for the current node's lock information.
+   */
+  public async getValueTimeLock (lockIdentifier: string): Promise<ValueTimeLockModel> {
```

Comment:
Following [TypeScript guidelines](https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines#null-and-undefined), We should avoid `null` in TypeScript whenever, so remove the mentioning of `null` in line 320; does this mean `lockIdnetifier` should be `string | undefined`?

Reply:
Refactoring it to 2 different methods.